index.html

```diff
@@ -1412,6 +1412,123 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID menthod, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      A reliable guide for making assessments of various technologies and weighing
+      ethical considerations is the 
+      <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        As with all things, strong consideration for the appropriate and ethical use
+        of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  
+        Avoidance of excess energy use that does not further other equally important 
+        goals such as human rights or personal privacy, particularly when that energy 
+        is sourced from non-sustainable energy creation methods, should be a best 
+        practice that serves the interest of all.
+      </p>
```

Comment:
```suggestion
      <p>
        When securing Decentralized Identifier systems, consideration should be given 
        to the full range of attack vectors, interdiction points, and centralization forces 
        vs the level of security required, or desired, to protect against them. Costs are 
        often associated with securing a wide array of systems, both digital and physical, 
        from computational systems to brick and mortar facilities. Decentralized Identifiers 
        may become the foundation for some of the most valuable activities in human life, 
        thus the level of security and protection against the harms outlined above should 
        be commensurate with the level of importance of what is being secured. 
      </p>
```

Reply:
`s/vs/relative to/`
`s/often associated/associated/`
`s/thus the level/and the level/`
`s/importance of what is/importance of the thing(s)/`

Reply:
But that's not really enough... @csuwildcat, your suggestion *completely* changes what's being said, eliminating the concern of energy consumption, which is not completely eliminated by switching to solar or wind or other generative methods because each has its own cost -- *and* by consuming those generative resources (including the resources involved in converting solar rays or wind to electricity, and getting that electricity to its point of consumption, *including* battery or other storage), you are potentially depriving other consumes of those resources, who are just trying to switch from non-renewable to renewable energy sources (a/k/a the general populace), *not* trying to add new consumption....  *sigh*